app/scripts/components/common/browse-controls/use-browse-controls.ts

```diff
@@ -48,7 +48,7 @@ export function useBrowserControls({ sortOptions }: BrowseControlsHookParams) {
   const [sortField, setSortField] = useQsState.memo(
     {
       key: Actions.SORT_FIELD,
-      default: sortOptions[0].id,
+      default: sortOptions[1]?.id || sortOptions[0]?.id, // If pubDate exists, default sorting to this
```

Comment:
Since the order of values on `sortOptions` may change and we're looking to have a specific default rather than the 1st, this could be made a bit more resilient.

```suggestion
      // If pubDate exists, default sorting to this
      default: sortOptions.find((o) => o.id === 'pubDate')?.id || sortOptions[0]?.id,
```